app/assets/javascripts/table.js

```diff
@@ -0,0 +1,125 @@
+function tableDraw()
+        {this.span();
+
+         var props = "";
+         var i = 0;
+         var dummyRows = this.content.length;
+         var dummyCols = this.content[0].length;
+
+         var dummyCode = '<table';
+         for (prop in this)
```

Comment:
Creating global 'for' variable. Should be 'for (var prop ...'.<br>The body of a for in should be wrapped in an if statement to filter unwanted properties from the prototype.